extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss

```diff
@@ -1,113 +1,90 @@
-@import 'skins/shared/mixins/clearfix';
 @import 'skins/shared/color';
+@import 'skins/shared/mixins/box-sizing';
+@import 'skins/oasis/css/core/responsive-variables';
 
 $branding-color: #000;
+$global-footer-color-border: #c7d1d8;
+$global-footer-vertical-spacing: 30px;
 
 .global-footer {
-	box-sizing: content-box;
-	clear: left;
-	font-size: 11px;
-	padding-bottom: 10px;
-
-	nav {
-		@include clearfix;
-		margin-top: 15px;
-		position: relative;
+	@include box-sizing(border-box);
```

Comment:
According to http://caniuse.com/#feat=css3-boxsizing ```box-sizing``` works everywhere - you can safely remove include.